test/formats.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+cd scripts
+python alternate_bulk_formats.py
+cd -
+if [ $(git diff alternate_formats/* | wc -l) -eq 0 ]; then
+  echo "No alternate format changes needed..."
+else
+  echo "Alternate formats need regenerated.  Run alternate_bulk_formats.py"
```

Comment:
I do like having a sanity check script to make sure the files haven't gotten out of sync, but this approach won't actually catch bugs in `alternate_bulk_formats.py` itself, since it's using it to do the verification.

I think the ideal would be to have a script which, for each data file, opens up the YAML/CSV/JSON versions and does a "deep equal" approach to them to test for differences.